generate-release/src/release_notes.rs

```diff
@@ -1,17 +1,24 @@
 use anyhow::Context;
 
 use crate::{
-    github_client::GithubClient,
+    github_client::{GithubClient, GithubIssuesResponse, IssueState},
     helpers::{get_contributors, get_merged_prs},
 };
-use std::{collections::HashSet, io::Write as IoWrite, path::PathBuf};
+use std::{
+    collections::HashSet,
+    io::Write as IoWrite,
```

Comment:
```suggestion
    io::Write as _,
```

We never mention `IoWrite` explicitly, so we don't need to name it in the imports.

Reply:
```suggestion
    io::Write,
```